dae/dae/inmemory_storage/raw_variants.py

```diff
@@ -282,11 +283,11 @@ def filter_summary_allele(
         if variant_type is not None:
             if not variant_type.match([allele.allele_type]):
                 return False
-        if person_ids is not None:
-            if allele.is_reference_allele:
-                return False
-            if not set(allele.variant_in_members) & set(person_ids):
-                return False
+        # if person_ids is not None:
+        #     if allele.is_reference_allele:
+        #         return False
+        #     if not set(allele.variant_in_members) & set(person_ids):
+        #         return False
```

Comment:
I don't understand why you have commented this code out but if the code is no longer needed just delete it!

Reply:
Summary alleles do not support filtering by person IDs.

Fixed.